### ISSUE TITLE
Feature: Edit order of photos in collection

### ISSUE THREAD

**Issue by realpixelcode** [pixelfed/pixelfed#2575]
It should be possible to arrange the photos in a collection into a custom order.

**Comment by trwnh** [comment]
IMO a Collection should be chronological but it currently is not.

**Comment by cogdog** [comment]
Right now it's chronological, the oldest photo added is first. If anything, it should at least be reverse chronological.

Ideally, if I had my dreams, like Flickr Albums, an owner should be able to pick oldest first, newest first, random, or a manually sorted order.

**Comment by keithfancher** [comment]
> If anything, it should at least be reverse chronological.

I strongly agree. Came here to file a bug about this because it seemed like a mistake. Very strange that non-collection images are listed newest first and collections are the opposite. (Especially with no way to change the order.)

You can work around this by adding older images to your collection first, but if you ever want to add a new image to your collection you have to re-do *the entire thing* 😑
